src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+  action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+  method="post"
+  onsubmit="onSubmit()"
+ >
+      <!-- Personal info block -->
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input 
+          type="text" 
+          name="surname" 
+          id="surname"
+          autocomplete="off"
+          minlength="2"
+          maxlength="20"
+          >
+        </label>
+        <label for="name">
+          Name:
+          <input 
+          type="text" 
+          name="name" 
+          id="name"
+          autocomplete="off"
+          minlength="2"
+          maxlength="20"
+          >
+        </label>
+        <label for="age">
+          How old are You?
+          <input 
+          type="number" 
+          name="age" 
+          id="age" 
+          value="12"
+          min="10"
+          max="100"
+          >
+        </label>
+        <label for="birth">
+          Full date of birth:
+          <input 
+          type="date" 
+          name="birth" 
+          id="birth"
+          
+          >
+        </label>
+        <label for="terms-of-agreement">
+          I accept the term of the agreement
+          <input 
+          type="checkbox" 
+          name="terms-of-agreement" 
+          id="terms-of-agreement"
+          required
+          >
+        </label>
+      </fieldset>
+      <!-- Registration block -->
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email">
+          E-mail:
+          <input 
+          type="email" 
+          name="email" 
+          id="email" 
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label for="password">
+          Password:
+          <input 
+          type="password" 
+          name="password" 
+          id="password"
+          required
+          >
+        </label>
+      </fieldset>
+      <!--Interesting facts block -->
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label for="cats">     
+          Do you love cats?
+          <input 
+          type="radio" 
+          name="cats" 
+          value="yes" 
+          id="cats-yes"
+          ><label for="cats-yes" class="radio">Yes</label>
+          <input 
```

Comment:
```suggestion
          <input 
            type="radio" 
            name="cats" 
            value="yes" 
            id="cats-yes"
          >
          <label for="cats-yes" class="radio">
            Yes
          </label>
          <input 
```
Нові теги повинні починатись з нових рядків